### DISCUSSION TITLE
Is there has any method to support VS 2022

### DISCUSSION THREAD

**Discussion by EagleTsui** [vuejs/language-tools#2357] [Q&A]
Currently I use VS 2022 to develop my website and with ASP.net with Vue 3.  I want syntax checking tools like this for .vue file.  May I know Is there has any method to support VS 2022 or has author any plan to support VS 2022 or above?

**Comment by johnsoncodehk** [comment]
You can download .vsix at https://marketplace.visualstudio.com/items?itemName=Vue.volar and install you self: https://learn.microsoft.com/en-us/visualstudio/ide/finding-and-using-visual-studio-extensions?view=vs-2022#install-without-using-the-manage-extensions-dialog-box

**Reply by EagleTsui** [reply]
I try to download vsix file from marketplace and duplicate click the vsix file to install it but I cannot be installed.  The following is install log

20/2/2023 9:39:23 am - -------------------------------------------
20/2/2023 9:39:23 am - Microsoft VSIX Installer
20/2/2023 9:39:23 am - -------------------------------------------
20/2/2023 9:39:23 am - Initializing Install...
20/2/2023 9:39:23 am - Extension Details...
20/2/2023 9:39:23 am - 	Identifier         : volar
20/2/2023 9:39:23 am - 	Name               : Vue Language Features (Volar)
20/2/2023 9:39:23 am - 	Author             : Vue
20/2/2023 9:39:23 am - 	Version            : 1.1.4
20/2/2023 9:39:23 am - 	Description        : Language support for Vue 3
20/2/2023 9:39:23 am - 	Locale             : en-US
20/2/2023 9:39:23 am - 	MoreInfoURL        : 
20/2/2023 9:39:23 am - 	InstalledByMSI     : False
20/2/2023 9:39:23 am - 	SupportedFrameworkVersionRange : [0.0,2147483647.2147483647]
20/2/2023 9:39:23 am - 
20/2/2023 9:39:23 am - 	SignatureState     : Unsigned
20/2/2023 9:39:23 am - 	Supported Products : 
20/2/2023 9:39:23 am - 		Microsoft.VisualStudio.Code
20/2/2023 9:39:23 am - 			Version : 
20/2/2023 9:39:23 am - 			ProductArchitecture : x86
20/2/2023 9:39:23 am - 
20/2/2023 9:39:23 am - 	References         : 
20/2/2023 9:39:23 am - Signature Details...
20/2/2023 9:39:23 am - 	Extension is not signed.
20/2/2023 9:39:23 am - 
20/2/2023 9:39:23 am - Searching for applicable products...
20/2/2023 9:39:23 am - Found installed product - Visual Studio Enterprise 2022
20/2/2023 9:39:23 am - Found installed product - Global Location
20/2/2023 9:39:23 am - VSIXInstaller.NoApplicableSKUsException: One or more extensions are for Visual Studio Code. Try installing them in Visual Studio Code.
   at VSIXInstaller.ExtensionService.GetInstallableDataImpl(IInstallableExtension extension, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionService.GetInstallableData(String vsixPath, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionPackService.IsExtensionPack(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.ExtensionPackService.ExpandExtensionPackToInstall(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()

**Reply by johnsoncodehk** [reply]
Just had a look at how the VS extension is maintained, unfortunately the workload is too heavy for us, so there are no plans to support and maintain it for now.
